drud/ddev
# Support php 7.3

## The Problem/Issue/Bug:

PHP 7.3 will be delivered December 13, 2018, and I'm sure our community will want it. 

The current version in this PR is "PHP 7.3.0RC5 (cli) (built: Nov 12 2018 13:37:51)", we'll want to make sure the actual released version gets into the final build for release v1.5.0

## How this PR Solves The Problem:

* Add php7.3 to the container
* Allow 7.3 config in ddev
* Bump the version to use the pushed ddev-webserver

## Manual Testing Instructions:

* Configure a site for php7.3
* Visit the site, look at phpinfo (or on Drupal, admin/reports/status/php
* Use php --version to verify version.


## Automated Testing Overview:

* Added 7.3 in container tests

## Related Issue Link(s):

## Release/Deployment notes:

Note that php7.3 does not yet support php-memcached. It's unknown whether memcached will be ready at release, see https://github.com/php-memcached-dev/php-memcached/issues/408. This means we have to let our users know, and also re-enable php7.3-memcached whenever it *does* become ready. (Currently there's special handling in the Dockerfile to exclude php7.3-memcached)
